docs/building-blocks.md

```diff
@@ -309,6 +309,28 @@ SparkleFormation.new(:instance_stack) do
 end
 ~~~
 
+Additional parameters can be passed into registry methods
+
+~~~ruby
+SfnRegisty.register(:add_foo_bar) do |args={}|
```

Comment:
doh, apparently I was copying/pasting out of my code. Thanks for the correction (although given #153 I'll close out the branch unfixed :)